projects/packages/my-jetpack/_inc/components/product-cards-section/protect-card/style.scss

```diff
@@ -0,0 +1,53 @@
+.value-section {
+    display: flex;
+    justify-content: space-between;
+
+    &__heading {
+        font-size: 12px;
```

Comment:
Sorry I didn't catch this last time, it looks like we can use `--font-body-extra-small` for this 🤔